test/common/schemaform/review/01-blank-form.e2e.spec.js

```diff
@@ -0,0 +1,59 @@
+const E2eHelpers = require('../../../e2e/e2e-helpers');
+const Timeouts = require('../../../e2e/timeouts.js');
+const HcaHelpers = require('../../../e2e/hca-helpers.js');
+// const testData = require('../../../hca/schema/maximal-test.json');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    HcaHelpers.initApplicationSubmitMock();
+
+    // Ensure introduction page renders.
+    client
+      .url(`${E2eHelpers.baseUrl}/health-care/apply/application/review-and-submit`)
+      .waitForElementVisible('body', Timeouts.normal)
+      .assert.title('Apply for Health Care: Vets.gov')
+      .waitForElementVisible('.schemaform-title', Timeouts.slow);  // First render of React may be slow.
+
+    E2eHelpers.overrideVetsGovApi(client);
+    E2eHelpers.overrideSmoothScrolling(client);
+
+    // Open first panel
+    client
+      .waitForElementVisible('.usa-accordion-bordered', Timeouts.normal)
+      .click('button.usa-button-unstyled')
+      .pause(1200);
+    client.expect.element('.edit-btn').to.be.visible;
+
+    // Try to save form with validation errors
+    client
+      .waitForElementVisible('.edit-btn', Timeouts.normal)
+      .click('.edit-btn');
+    client.expect.element('input[name="root_veteranFullName_first"]').to.be.visible;
+
+    client
+      .waitForElementVisible('.form-review-panel-page .usa-button-primary', Timeouts.normal)
+      .click('.form-review-panel-page .usa-button-primary');
```

Comment:
This test is failing here; sometimes the button gets clicked, and sometimes it doesn't, causing the next line that checks for the error class to be present to fail.

Reply:
Could repeating the click help to ensure that the button gets clicked without disrupting the test?